### PR TITLE
Add RFC 79: Listing of Service Providers on GDAL website

### PR DESCRIPTION
**Pull request by rouault**

### CODE REVIEW

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Core Contributors are essentially the organizations that have one or more
+Core Committer (a person with merge rights in the GDAL GitHub repository)
+and/or PSC members as part of their staff and who provide
+GDAL related services to customers.
+
+To qualify, the Core Committers and/or PSC members must be directly employed
+by the organization and not simply part time contractors.
+
+They get to add a 200px wide logo with a decriptive text outlining their qualifications and services.
```
Comment (mloskot) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:65]: `s/decriptive/descriptive`

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+2.2 Contributors
+----------------
+
+Contributors are organizations who provide GDAL related services and
+have one or more people on their staff who are well known in the GDAL
+community for their long term contributions to the project. Contributions are
+not limited to source code and can also be in the form of documentation,
+binary builds, active user support via the public forums, etc.
+
+They get to add a 150px wide logo with a decriptive text outlining their qualifications and services.
```
Comment (mloskot) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:76]: `s/decriptive/descriptive`

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+2.3 Other Service Providers
+---------------------------
+
+Organization with GDAL expertize and providing services around GDAL
+can fit in this category. While proprietary solutions providers are welcome,
+we also expect organizations listed here to be providing legitimate services to
+users of the Open Source software as well and don't want this
+to just become an advertizement board for proprietary products.
+
+They get to add a 100px wide logo with a decriptive text outlining their qualifications and services.
```
Comment (mloskot) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:87]: `s/decriptive/descriptive`

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+They get to add a 150px wide logo with a decriptive text outlining their qualifications and services.
+
+2.3 Other Service Providers
+---------------------------
+
+Organization with GDAL expertize and providing services around GDAL
+can fit in this category. While proprietary solutions providers are welcome,
+we also expect organizations listed here to be providing legitimate services to
+users of the Open Source software as well and don't want this
+to just become an advertizement board for proprietary products.
```
Comment (mloskot) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:85]: `s/advertizement/advertisement`

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+They get to add a 100px wide logo with a decriptive text outlining their qualifications and services.
+
+2.4 Refreshing the service providers list
+-----------------------------------------
+
+If at some point (every 2 years?) we feel that the list is or might be no
+longer current, we could bring it up to date as follows:
+
+* create a github ticket dedicated for the refresh
```
Comment (mloskot) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:95]: `s/github/GitHub/`

**Review by mloskot — CHANGES_REQUESTED** [review]
Picked a few typos

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+community for their long term contributions to the project. Contributions are
+not limited to source code and can also be in the form of documentation,
+binary builds, active user support via the public forums, etc.
+
+They get to add a 150px wide logo with a descriptive text outlining their qualifications and services.
+
+2.3 Other Service Providers
+---------------------------
+
+Organization with GDAL expertize and providing services around GDAL
```
Comment (EvertEt) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:81]: Minor: I believe this should be expertise

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+================
+
+A new Service Providers page will be added to the Sphinx documentation.
+
+The service providers will be grouped in three categories. The descriptions
+below should make it relatively easy to determine in which group a given
+organization belongs.
+
+Each qualified organization who wishes to be listed is responsible for
+adding themselves to the right category in the page through a pull
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:35]: right -> correct

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+request (or a direct commit in case of core committers). We are intentionally
+not going to provide instructions here for preparing the pull request as
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:37]: going to provide -> providing

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+it is expected that a qualified service provider should have people on
+staff who can figure it out themselves. Yes, we intentionally
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:39]: drop "themselves"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+Organizations in each category are displayed in randomized order (except
+in the PDF docs which are static). However entries should be added to the
+source file in alphabetical order (for PDF docs purpose).
+The output will be similar to
+https://mapserver.org/community/service_providers.html
+
+2.1 Core Contributors
+---------------------
+
+Core Contributors are essentially the organizations that have one or more
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:57]: Drop "essentially the"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+Core Committer (a person with merge rights in the GDAL GitHub repository)
+and/or PSC members as part of their staff and who provide
+GDAL related services to customers.
+
+To qualify, the Core Committers and/or PSC members must be directly employed
+by the organization and not simply part time contractors.
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:63]: Drop "simply"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+
+They get to add a 200px wide logo with a descriptive text outlining their qualifications and services.
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:65]: Should there be a cap for the max height here and below?

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+binary builds, active user support via the public forums, etc.
+
+They get to add a 150px wide logo with a descriptive text outlining their qualifications and services.
+
+2.3 Other Service Providers
+---------------------------
+
+Organization with GDAL expertise and providing services around GDAL
+can fit in this category. While proprietary solutions providers are welcome,
+we also expect organizations listed here to be providing legitimate services to
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:83]: Drop "also"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+users of the Open Source software as well and don't want this
+to just become an advertisement board for proprietary products.
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:85]: "well and don't want this to just become an advertisement" -> "well. This is not  an advertisement"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+
+If at some point (every 2 years?) we feel that the list is or might be no
+longer current, we could bring it up to date as follows:
+
+* create a GitHub ticket dedicated for the refresh
+* send an email to the list with this message "in 4 weeks, we are going to remove
+  all currently listed service providers to refresh the site. If you are still
+  interested, add a note to ticket XXXX".
+
+Instead of the ticket we could just ask to reply to the email but the advantage of
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:100]: "people to reply to the email, but"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+the ticket is to generate less email traffic for those not interested, plus it
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:101]: "interested. It"

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+leaves a trace of the list of organizations who responded in a single place inside the ticket.
+
+3. FAQ
+======
+
+* Should we also recognize sponsors or organizations that are funding significant
+  features in this page?
+
+  * No, since this is a service provider page and not a sponsorship page. However
```
Comment (schwehr) [inline gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst:110]: "No, since this" -> "No. This"

**Review by schwehr — COMMENTED** [review]
Minor suggestions

File: gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst (hunk 12 above)
Comment (rouault) [reply gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst]: I've changed to "200 px max high or wide logo" and similarly for the other categories

**Pull request merged**